README.md

```diff
@@ -57,13 +57,13 @@ To join us in these efforts, please feel free to reach out, raise issues or read
 
 ## Installation 🎬
 
-You can install the latest release of Mava for **Tensorflow** from PyPI (to be deprecated soon). Note that you will still be able to use TF2 systems by pinning to an older version of Mava.
+You can install the latest release of Mava as follows:
 
 ```bash
-pip install id-mava[reverb,tf,envs]
+pip install id-mava[reverb,jax,envs]
```

Comment:
Currently, we don't have a release for Jax, so this won't work, right? Or at least won't install any working system. If so, are we planning to only merge in this PR once we have finished the initial Jax release? Alternatively, I guess we can get this PR in, but then only have the source install as an option for now?

Reply:
So this actually works since I added JAX to the setup file before our prev release.